outlines/debugging.md

```diff
@@ -0,0 +1,17 @@
+1. Using the javascript `debugger` keyword
```

Comment:
I'd say that methods like logging to the console and using Meteor shell to call code are simpler than diving into the JS debugger and stepping through code (personally, I only use the debugger as a last resort). Perhaps we should start with those before hitting the super heavy duty tool?